src/pds/ingress/pds-nucleus-datasync-completion.py

```diff
@@ -36,45 +38,36 @@ def lambda_handler(event, context):
 
     logger.info(f"Lambda Request ID: {context.aws_request_id}")
     logger.info(f"Event: {event}")
-    resource = event['resources']
 
-    resource_list = resource[0].split("/")
-    task_id = resource_list[1]
-    exec_id = resource_list[3]
+    json_event = json.dumps(event)
+    transfer_report = json.loads(json_event)
```

Comment:
Since `json.loads` is just the inverse of `json.dumps`, why not just use `json_event` directly? This seems like a no-op … is there an ulterior motive? 🤷

Reply:
@nutjob4life , Thanks for catching that! That event earlier had some text formatting that was not accepted as proper JSON format and the lambda was rejecting the event. So I sent the event through `json.dumps` and `json.loads` to automatically format JSON it and it worked. However, this is not required anymore and the current event format is accepted by lambda. So I removed those 2 lines and updated the code to directly process the event.

Reply:
Ahh okay! Thanks!